docs/pages/guides/analyzing-bundles.mdx

```diff
@@ -10,77 +10,39 @@ Bundle performance varies for different platforms. For example, web browsers don
 
 ## Analyzing bundle size
 
-The libraries used in a project influence the size of the production JavaScript bundle. You can use [source map explorer](https://www.npmjs.com/package/source-map-explorer) to visualize the production bundle and identify which libraries are contributing to the bundle size.
+The libraries used in a project influence the size of the production JavaScript bundle. Starting from **Expo SDK 51**, you can use [Expo Atlas](https://github.com/expo/expo-atlas#readme) to visualize the production bundle and identify which libraries contribute to the bundle size.
 
-<Step label="1">
+### Using Atlas with `expo start`
```

Comment:
I was on the fence of this one. It's a bit verbose to have in the title itself, here we just reference the "start command" (as `expo start`). We also list out the full command in the terminal block here.

No strong opinions though, if you think we should do this, let's add it.

Reply:
Yeah, I can understand. The reason I suggested using the `npx` prefix is because the `npx expo start` is like a universal command that we say everywhere. It becomes easier for the user to connect the dots as well as quickly review this doc. Also, another thing to note here is that `expo start` (even through that's not the intention) is somewhat relates to the older CLI iirc. We can avoid any room for confusion when onboarding a user persona who might be familiar with our tools updates but at the same might not be too up to date.